docs/components/validering/FValidationForm.md

```diff
@@ -29,6 +29,31 @@ Då finns ett alternativ att fokusera på första felaktiga fältet.
      </f-validation-form>
 ```
 
+## Formulär med dolda fält
+
+Om man använder sig av FErrorList och har inmatningsfält i komponenter som kollapsar, t.ex. FExpandablePanel, så måste man använda `errorListBeforeNavigate` för att öppna containern innan fokus kan flyttas till inmatningsfältet.
+
+```diff
+     <f-validation-form
+         @submit="onSubmit"
++        :error-list-before-navigate="expandPanel"
+     >
+      <template #default>
+        <f-expandable-panel :expanded="expand">
+          <template #title>Panel to be expanded</template>
+           <template #default>
+            <f-text-field
+              id="field1"
+              v-model="field1"
+              v-validation.required
+            >
+                Field1
+            </f-text-field>
+          </template>
+        </f-expandable-panel>
+     </f-validation-form>
+```
+
```

Comment:
Kan vi lägga in `WithErrorListAndCbFunction.vue` exemplet i dokumentationen så man ser något körbart